### PR TITLE
[MS] Adding Managed Disk as a Data Source

### PR DESCRIPTION
**Pull request by echuvyrov**
New data source - managed disks.

### CODE REVIEW

File: website/docs/d/managed_disk.html.markdown [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+* `resource_group_name` - (Required) Specifies the name of the resource group.
+
+
+## Attributes Reference
+
+* `storage_account_type` - The storage account type for the managed disk.
+* `source_uri` - The source URI for the managed disk
+* `source_resource_id` - ID of an existing managed disk that the current resource was created from.
+* `os_type` - The operating system for managed disk. Valid values are `Linux` or `Windows`
+* `disk_size_gb` - The size of the managed disk in gigabytes.
```
Comment (tombuildsstuff) [inline website/docs/d/managed_disk.html.markdown]: Could we add in the missing `tags` field too?

File: website/azurerm.erb [hunk 2]
```diff
@@ -25,6 +25,11 @@
                 <li<%= sidebar_current("docs-azurerm-datasource-public-ip") %>>
                     <a href="/docs/providers/azurerm/d/public_ip.html">azurerm_public_ip</a>
                 </li>
+
+                <li<%= sidebar_current("docs-azurerm-datasource-managed-disk") %>>
+                    <a href="/docs/providers/azurerm/d/managed_disk.html">azurerm_managed_disk</a>
+                </li>
```
Comment (tombuildsstuff) [inline website/azurerm.erb]: _minor_: can we sort this alphabetically in the list?

**Review by tombuildsstuff — APPROVED** [review]
Hey @echuvyrov 

Thanks for this PR - besides a couple of minor tweaks this LGTM :)

Tests pass:
```
$ TF_ACC=1 envchain azurerm go test ./azurerm -v -run TestAccDataSourceAzureRMManagedDisk -timeout 300m -parallel 5
=== RUN   TestAccDataSourceAzureRMManagedDisk_basic
--- PASS: TestAccDataSourceAzureRMManagedDisk_basic (197.60s)
PASS
ok  	github.com/terraform-providers/terraform-provider-azurerm/azurerm	197.611s
```

Thanks!

File: website/docs/d/managed_disk.html.markdown (hunk 1 above)
Comment (echuvyrov) [inline website/docs/d/managed_disk.html.markdown]: done

File: website/azurerm.erb (hunk 2 above)
Comment (echuvyrov) [inline website/azurerm.erb]: done

**Comment by echuvyrov** [comment]
@tombuildsstuff changes made, thanks for review!

**Comment by tombuildsstuff** [comment]
Changes LGTM - thanks @echuvyrov :)

**Pull request merged**